components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -10687,4 +10687,32 @@ public static String getClaimDisplayName(String claimURI, String subscriber) thr
         }
         return scopeToKeyMap;
     }
+
+    /**
+     * Extract Endpoint Registry Entry information
```

Comment:
shall we add this to an endpoint registry util class. if non exists, create one.

Reply:
ignore above